src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -87,6 +87,10 @@ public void setTestVerbosity(){
      * END OF LARGE FILES FOR TESTING
      */
 
+    public static final String NA12878_chr17_1k_BAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.bam";
+    public static final String NA12878_chr17_1k_CRAM = publicTestDir + "NA12878.chr17_69k_70k.dictFix.cram";
+    public static final String v37_chr17_1Mb_Reference = publicTestDir + "human_g1k_v37.chr17_1Mb.fasta";
```

Comment:
Why chr17? We are trying to stick to chromosomes 20 and 21 as much as possible so that we can use the references in the large files dir.